lib/hardware_control/photo_capturing/sony_remote_photo_capture.dart

```diff
@@ -15,19 +16,19 @@ class SonyRemotePhotoCapture extends PhotoCaptureMethod with UiLoggy {
   SonyRemotePhotoCapture(this.directoryPath);
 
   // Testing with camera gave ~165 ms with manual focus, ~220 with good autofocus
-  // and up to 500 ms in bad light. This should be avoided anyway because the (short)
-  // button press will not trigger the camera then. 
+  // and up to 500 ms in bad light. Differs per lens as well. 
+  // Short button presses will not trigger the capture when focussing is not complete.
   @override
-  Duration get captureDelay => Duration(milliseconds: 200);
+  Duration get captureDelay => Duration(milliseconds: SettingsManagerBase.instance.settings.hardware.captureDelaySony);
 
   void _capture() {
     loggy.debug("Sending capture command to Sony Remote");
     // AutoIt script line
     // https://ss64.com/nt/syntax-esc.html
-    var autoItScript = "ControlClick('Remote', '', 1001)";
+    var autoItScript = File('lib/hardware_control/photo_capturing/sony_remote_capture_script.au3');

```

Comment:
I have been thinking about this. Perhaps we should include the script as asset or setting. Then write it to %temp% on launch of the application (or on first creation of a Sony Remote Photo Capture class instance)

Reply:
Fixed